environment/src/main/java/com/sequenceiq/environment/credential/v1/converter/TelemetryRequestToTelemetryConverter.java

```diff
@@ -0,0 +1,36 @@
+package com.sequenceiq.environment.credential.v1.converter;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.cloud.model.Logging;
+import com.sequenceiq.cloudbreak.cloud.model.Telemetry;
+import com.sequenceiq.cloudbreak.cloud.model.WorkloadAnalytics;
+import com.sequenceiq.environment.api.v1.environment.model.request.LoggingRequest;
+import com.sequenceiq.environment.api.v1.environment.model.request.TelemetryRequest;
+import com.sequenceiq.environment.api.v1.environment.model.request.WorkloadAnalyticsRequest;
+
+@Component
+public class TelemetryRequestToTelemetryConverter {
```

Comment:
Please merge this and `TelemetryToTelemetryResponseConverter` into a single class, called `TelemetryApiConverter`